lib/aws/lex/conversation/type/sentiment_score.rb

```diff
@@ -7,27 +7,17 @@ module Type
         class SentimentScore
           include Base
 
-          required :positive, from: :Positive
-          required :negative, from: :Negative
-          required :neutral, from: :Neutral
-          required :mixed, from: :Mixed
+          required :mixed
+          required :negative
+          required :neutral
+          required :positive
 
-          class << self
-            def parse_string(str)
```

Comment:
It's nice that we no longer need to build a custom parser for the sentiment score string.

Reply:
that was a really nice change to the API